### DISCUSSION TITLE
Clear Chat di Twich

### DISCUSSION THREAD

**Discussion by Peppe-73** [FortAwesome/Font-Awesome#19706] [General]
Hi, Twitch Clear Chat is not working yet get version 2.1.3

**Comment by tagliala** [comment]
Hi, I think this message does not belong to Font Awesome

**Comment by Peppe-73** [comment]
It seems so to me, I took it from here, see screenshots

https://imgbox.com/jYo7EOzb

**Reply by tagliala** [reply]
Thanks for the clarification. We  do not provide support to third parties, please contact the author of "Clear Chat"
